package.json

```diff
@@ -6,7 +6,12 @@
   "scripts": {
     "develop": "webpack-dev-server --mode development",
     "build": "webpack --mode production",
-    "build:watch": "webpack --watch --mode production"
+    "build:watch": "webpack --watch --mode production",
+    "prettier:write": "yarn prettier --write .",
```

Comment:
Not a big deal, but should this include `--ignore-unknown` to match the lint-staged rule?  I'm not 100% sure if it matters though.